client/signup/steps/welcome/index.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+
+class WelcomeStep extends Component {
+	state = {
```

Comment:
It doesn't look like this component is actually using any state variables so maybe we can remove this?